cpp/tensorrt_llm/kernels/trtllmGenKernels/blockScaleMoe/RoutingRenormalize.cu

```diff
@@ -23,14 +23,18 @@ namespace routingRenormalize
 
 static constexpr int NumThreads = 1024;
 static constexpr int NumWarps = NumThreads / WarpSize;
-static constexpr int MaxNumTopExperts = 8;
-static constexpr int MaxNumExperts = 128;
+static constexpr int MaxNumTopExperts = 10;
```

Comment:
for kernels/models which only require 8 max experts, moving to max 10 experts only increases the number of required experts, right?